features/all-positions/PositionActionsDialog.tsx

```diff
@@ -298,7 +302,10 @@ const PositionActionsDialog = (props: DialogProps) => {
         setHash(null);
         setSuccess(null);
         setError(null);
-        const collateralToDepositWei = toWei(collateralToDeposit);
+        const collateralToDepositWei = toWei(
```

Comment:
Throughout this PR I round all numbers before calling `toWei` on them because its possible that the string to convert `toWei` has more decimals than the number in `wei` can convert, which throws an error. Example, converting `toWei(0.123456789, 8)` will throw an error because there are 9 decimals but the precision specified is 8.

I hard code 6 as a safe precision value meaning this dApp will work for all tokens with >= 6 precision